lib/bookingsync-engine.rb

```diff
@@ -1 +1,29 @@
-require 'bookingsync'
+require "bookingsync"
+
+module BookingSyncEngine
+  cattr_accessor :support_multi_applications
+  self.support_multi_applications = false
+
+  cattr_accessor :single_app_model
+  self.single_app_model = -> { ::Account }
```

Comment:
why is it a lambda and not a class directly?

Reply:
In the case the class is not loaded when the initializer is.

Reply:
Alternatively, could be a string, but I guess lambda gives more options